run_tests.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * This script runs tests for all packages in `./packages` and
+ * example projects in `./examples`.
+ */
+
+const fs = require('graceful-fs');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const PACKAGES_DIR = './packages';
+const EXAMPLES_DIR = './examples';
+
+const packages = fs.readdirSync(PACKAGES_DIR)
+  .map((file) => path.resolve(PACKAGES_DIR, file))
+  .filter((f) => fs.lstatSync(path.resolve(f)).isDirectory());
+
+const examples = fs.readdirSync(EXAMPLES_DIR)
+  .map((file) => path.resolve(EXAMPLES_DIR, file))
+  .filter((f) => fs.lstatSync(path.resolve(f)).isDirectory());
+
+
+packages.forEach((dir) => {
```

Comment:
the code style for Jest is not to use parenthesis for single identifier arguments to arrow functions. Also, I recommend just calling this `cwd` so you can use the short notation below `{cwd, stdio: [0, 1, 2]}`.